picard/formats/mp4.py

```diff
@@ -182,7 +192,13 @@ def _save(self, filename, metadata):
             elif name in self.__r_int_tags:
                 try:
                     tags[self.__r_int_tags[name]] = [int(value) for value in values]
-                except ValueError:
+                except (ValueError, TypeError) as e:
```

Comment:
It's not clear to me why a `TypeError` would be raised here.

Reply:
I received a TypeError during testing so added it as an exception.